src/components/share-overlay/share-overlay.js

```diff
@@ -674,7 +674,7 @@ class ShareOverlay extends Component {
               isIos={this.isIos}
               videoClippingOption={this.state.videoClippingOption}
             />
-            {this._renderVideoClippingOptions()}
+            {this.props.activePresetName !== ReservedPresetNames.MiniAudioUI && this._renderVideoClippingOptions()}
```

Comment:
this solution will work if we only want to have 1 extra special preset which we want to inject data from plugins.
once we have 2+, this will not be scalable.

if we can't come up with something this version we can keep it like this i guess, but we should at least mark this code down as something that will need refactor.

Reply:
Initially, i wanted to do the change from within the audio plugin (via config) but @MosheMaorKaltura  suggested that, that way is better as it eliminates any potential misconfiguration  , anyway our preset number is very limited